pkg/virt-handler/vm.go

```diff
@@ -98,6 +98,14 @@ func (d *VMHandlerDispatch) updateVMStatus(vm *v1.VM, cfg *api.DomainSpec) error
 		return err
 	}
 	vm = obj.(*v1.VM)
+
+	// XXX When we start supporting hotplug, this needs to be altered.
+	// Check if the VM is already marked as running. If yes, don't update the VM.
+	// Otherwise we end up in endless controller requeues.
```

Comment:
I guess this is where `.status.domain` will become helpful, not?

It would help us to chekc if the VM was updatde (the object) or if the _domain changed_ (hotplug).